src/eval-frame/components/panes/history-item.jsx

```diff
@@ -45,16 +50,14 @@ export class HistoryItemUnconnected extends React.Component {
       case "CONSOLE_OUTPUT": {
         return (
           <ConsoleMessage level={this.props.level || "OUTPUT"}>
-            <ValueRenderer valueToRender={this.props.valueToRender} />
+            <ValueRenderer valueKey={this.props.historyId} />
           </ConsoleMessage>
         );
       }
       case "FETCH_CELL_INFO": {
         return (
           <ConsoleMessage level={this.props.level || "OUTPUT"}>
-            <PreformattedTextItemsHandler
```

Comment:
PreformattedTextItemsHandler was overkill. we can just render the fetch cell info as a single <pre> string, which has advantages in many places